app/forms/email_address_form.rb

```diff
@@ -2,6 +2,9 @@ class EmailAddressForm < Form
   attribute :email_address
   attribute :resend, :boolean
 
+  # Max length is based on the lowest requirement on services Claim interacts, in this case Payroll
+  # https://www.gov.uk/government/publications/real-time-information-internet-submissions-2024-to-2025-technical-specifications
+
```

Comment:
Suggestion: Move this comment to the config file where `email_max_length` is defined.